yarn.lock

```diff
@@ -313,13 +313,13 @@
     fuzzy "^0.1.3"
     glob "7.1.4"
     inquirer "^8.2.5"
-    inquirer-checkbox-plus-prompt "^1.0.1"
+    inquirer-checkbox-plus-prompt "^1.4.2"
     js-yaml "3.13.1"
     ora "5.4.1"
     proxy-agent "5.0.0"
     rimraf "^3.0.2"
     semver "^7.3.7"
-    simple-git "3.15.0"
+    simple-git "3.16.0"
```

Comment:
should `@datadog/datadog-ci` use `^3.16.0` instead 🤔

Reply:
updated!